.github/workflows/pull_request_verify.yaml

```diff
@@ -0,0 +1,172 @@
+name: Pull Requests
+
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+env:
+  CARGO_TERM_COLOR: always
+  RUST_BACKTRACE: full
+  CARGO_TARGET_DIR: ${{ github.workspace }}/../target
+  LOGBAK_USER: ${{secrets.LOGBAK_USER}}  #LOCBAK_* for upload logs to server when test failed, for windows
+  LOGBAK_PASSWORD: ${{secrets.LOGBAK_PASSWORD}}
+  LOGBAK_SERVER: ${{secrets.LOGBAK_SERVER}}
+jobs:
+#job for checkout the ckb
+  checkout:
+    name: ckb-checkout
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [Linux-runner, mac-runner,win-runner]
+        include:
+          - os: Linux-runner
+            script_run: make build
+          - os:  mac-runner
+            script_run: make build
+          - os: win-runner
+            script_run: devtools/windows/make prod
+    steps:
+    - name: Set Cargo path
+      run: echo "$HOME/.cargo/bin" >> $GITHUB_PATH
+    - uses: actions/checkout@v2
+      with:
+        submodules: false
+        fetch-depth: 2
+    - run: ${{ matrix.script_run }}
```

Comment:
This job named "checkout". Then why it executes `run: ${{ matrix.script_run }}`?

Reply:
And why "checkout" needs `strategy.matrix`

Reply:
The checkout also include build steps, build windows and ubuntu/mas has different command.

Reply:
removed build steps.